iothub_client/samples/iothub_client_sample_mqtt/iothub_client_sample_mqtt.c

```diff
@@ -213,8 +213,9 @@ void iothub_client_sample_mqtt_run(void)
     }
 }
 
-int main(void)
+int main(int argc, char* const argv[])
```

Comment:
for the samples we do not expose getting the arguments from command line.
Let's keep the standard used across the other samples and read it explicitly from the static const char* connectionString variable.

Reply:
It has some disadvantages, if the connection string must be hard coded in the sources.
Especially, if the host which runs the binaries is not identical with the build machine which could run the cross-compiler.
Therefore I have added this changes.